conf/jhove/jhove.conf

```diff
@@ -3,7 +3,7 @@
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xmlns="http://hul.harvard.edu/ois/xml/ns/jhove/jhoveConfig"
  xsi:schemaLocation="http://hul.harvard.edu/ois/xml/ns/jhove/jhoveConfig
-                     file:///usr/local/glassfish4/glassfish/domains/domain1/config/jhoveConfig.xsd">
+                     file:///usr/local/payara5/glassfish/domains/domain1/config/jhoveConfig.xsd">
```

Comment:
This has been fine under Glassfish4 and is fine under Payara5, but installation-main sez

> The installer places files called jhove.conf and jhoveConfig.xsd into the directory /usr/local/glassfish4/glassfish/domains/domain1/config by default and makes adjustments to the jhove.conf file based on the directory into which you chose to install Glassfish.

We might a) update that to payara5 and b) I don't see logic in the script to adjust the path in jhove.conf; we might note in the docs that the file would need to be edited?

Reply:
There's no code in the script that adjusts that path, no. But there's **the next best thing** in there, a comment:
```
# @todo: The JHOVE conf file has an absolute PATH of the JHOVE config schema file (uh, yeah...)
# - so it may need to be readjusted, if glassfish lives somewhere other than /usr/local/glassfish4
# (replicate from the old installer)
```
(it looks like it just slipped through the cracks. will add.)